main/getting-started/alpha.md

```diff
@@ -0,0 +1,60 @@
+# Agoric Alpha	
+Release Version: Adriatic
+
+![Alpha Cover](./assets/alpha-cover.png)
+
+We’re building components to power the DeFi ecosystem. Come take a peek under the hood and help us scale our next-gen infrastructure. The Alpha Adriatic release provides you access to our toolset, API, and public chain.
+
+[Start Coding](https://agoric.com/documentation/getting-started/)
+[Join Community](https://discord.gg/gC9z6US)
+
+---------------------
+
+## What you can build in Alpha Adriatic 
+
+Numerous transaction components -- what we think of as DeFi Legos -- are available in the Alpha Adriatic release. We’ll be adding to the pool with our growing community’s help and participation in the coming months as we scale contributions. The initial slate of elements includes the following: 
+
+![Automated Market Maker](./assets/amm.png)
+### Automated Market Maker
+Build a smart contract that acts as an on-chain automated DEX with liquidity. 
+
+![Autoswap](./assets/autoswap.png)
+### Autoswap
+Tap into our Uniswap-like, constant-product AMM that executes upon finding a match in the current liquidity pool.
+
+![Covered Call](./assets/covered-call.png)
+### Covered Call
+Create an option contract that enables the holder to buy a specific, previously escrowed asset at a predetermined strike price (before expiration).
+
+![Simple Exchange](./assets/simple-exchange.png)
+### Simple Exchange
+A basic exchange with an order book for one asset, priced in a second asset.
+
```

Comment:
Do we want to add a final ### Additional Sample Components
"We also provide examples of:  (list of just names)". I know we're got more than just these.